kube/match/match-service.yaml

```diff
@@ -0,0 +1,15 @@
+# Match service that exposes the go endpoint to the cluster
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: match
+  name: match
+spec:
+  ports:
+  - name: "api"
```

Comment:
Also name here doesn't match the `user-service` equivalent one

Reply:
And is there a reason why this is in quotes, when nothing else is?